multilateralfund/settings.py

```diff
@@ -35,7 +35,11 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = [
+    'multilateralfund.edw.ro',
+    'localhost',
+    '127.0.0.1'
+]
```

Comment:
I would highly recommended not hard-coding these and loading them from the env instead. We don't want any of these hosts allowed in a prod environment for example.